lib/polyphony/net.rb

```diff
@@ -35,9 +35,10 @@ def socket_from_options(host, port, opts)
         ::Socket.new(:INET, :STREAM).tap do |s|
           s.reuse_addr if opts[:reuse_addr]
           s.dont_linger if opts[:dont_linger]
+          s.reuse_port if opts[:reuse_port]
           addr = ::Socket.sockaddr_in(port, host)
           s.bind(addr)
-          s.listen(0)
+          s.listen(opts[:backlog] || Socket::SOMAXCONN)
```

Comment:
Having looked up the manpage for listen(2),  It is not clear to me what 0 backlog means. Anyways it's a good idea to stick with `SOMAXCONN` as default.